docs/en/source/post_queries.rst

```diff
@@ -0,0 +1,115 @@
+************
+POST QUERIES
+************
+
+The reason to provide a way to query NGSI_LD entities via POST, is using GET:
+
+1. We end up assembling very long URLs,due to URI parameters for 'id', 'attrs', etc, being included in the URL.
+2. By using POST, there is no need of long URLs.
+
+So, the difference lies in that instead of passing the inputs as URI parameters, we in POST Query service passes all the query items in the payload body.
```

Comment:
..., in a POST Query, we pass all the query items in the payload body.

Reply:
Done